src/components/H2.js

```diff
@@ -6,7 +6,7 @@ const styles = StyleSheet.create({
   subheadline: {
     fontSize: 22,
     marginBottom: 5,
-    fontFamily: fontFamilies.serifBold
+    fontFamily: fontFamilies.serifTitle
```

Comment:
the naming here might be a bit confusing but `serifBold` is the correct one for subheadlines. `serifTitle` is more of a logo font exclusively used for the (main/first) headline of an article.